catalog/systemd.catalog.in

```diff
@@ -119,6 +119,14 @@ Documentation: sd-login(3)
 
 A seat @SEAT_ID@ has been removed and is no longer available.
 
+-- b2bcbaf5edf948e093ce50bbea0e81ec
+Subject: The Secure Attention Sequence was pressed on @SEAT_ID@
+Defined-By: systemd
+Support: %SUPPORT_URL%
+Documentation: sd-login(3)
+
+The Secure Attention Sequence was pressed on @SEAT_ID@.
```

Comment:
maybe say what the sequence is. And what this typically does.

also, this is typcally called secure attention *key*. not sequence. it's not a sequence after all, you have to press the four keys at once...

Also, should have the abbreviation "SAK" somewhere, because that's a very recognizable one.

Reply:
Let me know if what I have here is good